# Problem 3
| | Part 1 | Part 2 |
|---|---|---|
| Rank | 3837 | 2306 |
| Time | 00:19:03 | 00:23:31 |

Today, I relearned that Python indices can be negative and not throw an error.

Yeah, that's about it. I spent literal minutes looking around the test input and actual input trying to figure out where I got extra cases from before remembering yesterday and thinking "Yeah, maybe my bug isn't where I think it is."

Screw Python.
